src/api/ingest/benthicpit_serializer.py

```diff
@@ -5,13 +5,17 @@
 from .serializers import CollectRecordCSVSerializer, build_choices
 
 
+__all__ = ["BenthicPITCSVSerializer"]
```

Comment:
Why is this necessary again? Without it, if you did `from benthicpit_serializer import *` wouldn't you get the same thing?

Reply:
the way it currently is yes